tests/test_misc.py

```diff
@@ -169,9 +169,11 @@ def test_unordered_dict(init_func):
 
 
 def test_topic_config_update():
+    # *NOTE* default.topic.config has been deprecated.
+    # This example remains to ensure backward-compatibility until it's removal
     confs = [{"message.timeout.ms": 600000, "default.topic.config": {"message.timeout.ms": 1000}},
              {"message.timeout.ms": 1000},
-             {"default.topic.config": {"message.timeout.ms": 1000}}]
```

Comment:
This should not change, since it tests that default.topic.config works.
Also, the change just creates a duplicate test case.

Reply:
doesn't the first case test that default.topic.config works?

Reply:
Why were there three tests to begin with then?
First test is: global + default
Second is: global
Third is: default

Reply:
I agree, so two tests it is

Reply:
No, we haven't removed any functionality yet, just deprecated it.
We still need to test all three combinations.

Reply:
ack